src/src/CustomBuild/ESPEasyDefaults.h

```diff
@@ -366,6 +366,9 @@
 #ifndef DEFAULT_SYSLOG_FACILITY
 #define DEFAULT_SYSLOG_FACILITY               	0 	    // kern
 #endif
+#ifndef DEFAULT_SYSLOG_PORT
+#define DEFAULT_SYSLOG_PORT                     0
```

Comment:
Should this new constant also be added to `Custom-sample.h`?

Reply:
Yeah that would be not such a bad idea I guess ;)